requirements.txt

```diff
@@ -1,6 +1,6 @@
 pip>=9.0.1
-numpy>1.16,<2.3
-sympy>=1.9,<1.14
+numpy>=2,<2.2.6
```

Comment:
Why the max numpy downgrade?

Reply:
Are there any systems that we know about where Devito is running and numpy is a system dependency (ie: air-gapped systems) where this might cause an issue?